### ISSUE TITLE
Suggestions for multi-line interactive mode

### ISSUE THREAD

**Issue by thewindgods** [aandrew-me/tgpt#321]
Hey, great software!
I would like to shamelessly ask for some features:

It would be nice if the multi-line interactive mode could allow you write prompts longer than 100 lines. I use Duckduckgo's AI services and it allows for far longer prompts when using the browser, so it would be great if tgpt would be able to do the same,

In addition it would also be fantastic if some more vi-keys were available in the multi-line interactive mode, like hjkl, G, gg, L, M, H to move around and potentially a visual mode too.

**Comment by santarl** [comment]
Would also love a shift + enter to submit

**Comment by amanshaw4511** [comment]
@thewindgods @santarl you could use multiline mode `tgpt -m`

**Comment by thewindgods** [comment]
@amanshaw4511 Yes, that's the mode that I and (presumably) @santarl are suggesting features for
